servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ReservableRequestConcurrencyControllers.java

```diff
@@ -207,12 +211,13 @@ public void onNext(@Nullable final ConsumableEvent<Integer> event) {
 
                 @Override
                 public void onError(final Throwable t) {
-                    LOGGER.info("Unexpected error from transportEventStream(MAX_CONCURRENCY).", t);
+                    LOGGER.info("{} Unexpected error from transportEventStream({})",
+                            connection, MAX_CONCURRENCY_KEY, t);
```

Comment:
I see two placeholders and three args here. Sadly I don't think we can use the `info(String, Throwable)` with other arguments.

Reply:
This is expected, `connection` and `MAX_CONCURRENCY_KEY` will go into placeholders, while `t` will be translated into stack-trace. If you look at the code before, it had zero placeholders and a single `t`.

Reply:
TIL. The code before used the `info(Object, Throwable)` method so it was obvious.